source/paramgen.go

```diff
@@ -19,13 +19,13 @@ func (Config) Parameters() map[string]sdk.Parameter {
 		},
 		"columns": {
 			Default:     "",
-			Description: "Comma separated list of column names that should be included in each Record's payload.",
+			Description: "todo: reove param, Comma separated list of column names that should be included in each Record's payload.",
```

Comment:
The todo comment shows up here. Not a biggie, if we're going to remove this before we release the next version.

Reply:
this was intentional cuz it won't be merged until the params are removed